sdk/src/ingredient.rs

```diff
@@ -1844,7 +1844,7 @@ mod tests_file_io {
         let ap = fixture_path("CIE-sig-CA.jpg");
         let ingredient = Ingredient::from_file(ap).expect("from_file");
         println!("ingredient = {ingredient}");
-        assert!(ingredient.validation_status().is_none());
+        assert!(ingredient.validation_status().is_some());
```

Comment:
This should not be reporting errors at the ingredient validation status level. what changed?

Reply:
It is identifying a CoseTimeStampMismatch on the nested manifest with the known error. I think that makes sense because this is a new check that would not have been performed when that nested image was added. So it is correct given that the image is now out of date. That's a useful test, but not what this test was designed to test. We should update the image that is saved in fixtures and this should not report an error anymore. 

```
  "validation_status": [
    {
      "code": "general.error",
      "url": "self#jumbf=/c2pa/contentauth:urn:uuid:5b639b8e-edc1-4e41-9c8e-c87fb1e36921/c2pa.signature",
      "explanation": "claim signature is not valid: CoseTimeStampMismatch"
    }
  ],
```